### ISSUE TITLE
Blank pages in front matter

### ISSUE THREAD

**Issue by mnjames** [sillsdev/ptx2pdf#616]
I have been using the following code to create blank pages in the front matter of our documents:
```
\periph |id="blank"
\ip \ 
```
(where the final \ has a space after it, to force a space to actually type onto the page so it's not considered completely blank by xetex.)

However, between 2.0.34 and 2.1.0 that code stopped working.

Was there a bug introduced? Or (more likely), is there a better method for me to add blank pages?

**Comment by davidg-sil** [comment]
Hmm. I've no idea if the `\[space]` is confusing something. A tilde (~) should be turned into a non-breaking space, though, or you could be more explicit about it and put in `\NBSP` (if paratext/your scripture editor lets you enter all-caps non-standard codes).

As a (somewhat tempramental) alternative, there's `\pb` (or its alias, `\pagebreak`).

**Comment by mnjames** [comment]
I've never gotten `\pb` or `\pagebreak` to work in the front matter. I think those are what I tried months ago before I came up with this hack-y workaround. If I remember correctly, they work in regular sfm files brought in from PT, but not in the front matter created directly in ptxPrint.

After a bit of experimenting I found the following works. It appears that all these elements are necessary to make xetex think the page has something written on it
`\ip ~\r`
This still seems ugly to me, but it *does* appear to work.
